ios/Fabric/RNCPagerViewComponentView.mm

```diff
@@ -408,7 +408,7 @@ + (ComponentDescriptorProvider)componentDescriptorProvider
 
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer {
 
```

Comment:
```suggestion
    if (!self.scrollEnabled) {
        return YES;
    }
```

Reply:
I guess, we should always recognize simultaneously in that case

Reply:
simultaneous by default [returns false](https://developer.apple.com/documentation/uikit/uigesturerecognizerdelegate/1624208-gesturerecognizer#return_value). So I wanted to preserve the original behaviour and allow recognize only when it's RN screen's triggered pop gesture. But i think your suggestion might also make sense e.g. current gesture is blocked so allow the other 🤔. Does this change fixes your issues?